onnx_caffe2/backend.py

```diff
@@ -501,6 +504,20 @@ def _direct_initialize_inputs(cls, inputs, initialized, ws, device_option):
             shape = list(d.dim_value for d in value_info.type.tensor_type.shape.dim)
             ws.FeedBlob(value_info.name, np.ones(shape), device_option)
 
+    @staticmethod
+    def optimize_onnx(input):
+        # this is where the binary is located when onnx-caffe2 has been installed
```

Comment:
you might need to modify it for internal BUCK-based build. Probably, try to import optimizer as a python module if it's installed and otherwise - fallback on the binary.

Reply:
ok, I'll be aware of that for when it comes up